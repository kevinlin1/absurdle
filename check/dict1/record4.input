argue
agnew
today
email
clash
slack
fifty
jiffy
