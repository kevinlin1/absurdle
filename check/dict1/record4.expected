2309 words
> argue
: ⬜⬜⬜⬜⬜

249 words
> agnew
: ⬜⬜⬜⬜⬜

162 words
> today
: ⬜⬜⬜⬜🟩

21 words
> email
: ⬜⬜⬜🟨⬜

8 words
> clash
: ⬜⬜⬜⬜⬜

2 words
> slack
: ⬜⬜⬜⬜⬜

2 words
> fifty
: 🟨🟩🟩⬜🟩

1 words
> jiffy
: 🟩🟩🟩🟩🟩

Absurdle 8/∞

⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜⬜⬜⬜🟩
⬜⬜⬜🟨⬜
⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
🟨🟩🟩⬜🟩
🟩🟩🟩🟩🟩
